build-system/tasks/build.js

```diff
@@ -72,6 +73,7 @@ async function performBuild(watch) {
         buildWebWorker({watch}),
         buildExtensions({watch}),
         compileAllUnminifiedTargets(watch),
+        compileExprs(),
```

Comment:
This might need to be chained before `buildExtensions` since extension code imports it.